src/java/htsjdk/samtools/reference/FastaSequenceFile.java

```diff
@@ -53,6 +54,13 @@ public FastaSequenceFile(final File file, final boolean truncateNamesAtWhitespac
         this.in = new FastLineReader(IOUtil.openFileForReading(file));
     }
 
+    /** Constructs a FastaSequenceFile that reads from the specified file. */
```

Comment:
old constructor should delegate to this one.  Possibly could benefit from adding an `IOUtil.nullFie(File file)` or something like that